src/control/server/storage/scm/provider.go

```diff
@@ -472,6 +473,12 @@ func (p *Provider) CheckFormat(req storage.ScmFormatRequest) (*storage.ScmFormat
 		res.Mountable = true
 	case fsTypeNone:
 		res.Formatted = false
+	case fsTypeUnknown:
+		// formatted but not mountable
+		p.log.Debugf("unexpected format of output from 'file -s %s'", req.Dcpm.Device)
+	default:
```

Comment:
From what I understand, the default case should never happen. Thus, an error should probably be returned.

Reply:
the default case will happen for any fs that is recognised but not one of the supported types e.g. there is a test case for this situation with ext2: https://github.com/daos-stack/daos/pull/7604/files/1a406044f09afb993b8c199b77ecbb6fc0769950#diff-c6c1b87e2db9b0eb1ce91d071551c308f10e18ec1583da175197c1f2ee1893d8R1058

Reply:
Thanks for the explanation, I misunderstood the default usage.
So everything seems OK for me :)